src/BloomBrowserUI/templates/customXMatter/MXBPamphlet-XMatter/MXBPamphlet-XMatter.less

```diff
@@ -25,25 +25,12 @@
 
 .insideBackCover {
     .marginBox {
-        .bloom-editable:only-child {
```

Comment:
The lack of comments in the original code here makes it somewhat impossible for a reviewer to know that most of what you are removing is no longer needed. The .Printer-Statement-style part, fine. But the rest, what can you tell us about what the old code was there and why it isn't needed?